UE/Application/Ports/TimerPort.cpp

```diff
@@ -22,11 +22,28 @@ void TimerPort::stop()
 void TimerPort::startTimer(Duration duration)
 {
     logger.logDebug("Start timer: ", duration.count(), "ms");
+    this->running = true;
+    this->timer = std::async(&TimerPort::timerThread, this, duration);
```

Comment:
`this` can be removed.
```suggestion
    running = true;
    timer = std::async(&TimerPort::timerThread, this, duration);
```